tests/cfb/test_pbp.py

```
@@ -588,8 +588,12 @@ def test_yards_per_drive():
 )
 def test_25_yardage_detection(play_text: str, yards_field: str, expected_yards: int):
     test = CFBPlayProcess(gameId = 401754571)
-    test.espn_cfb_pbp()
-    json_dict_stuff = test.run_processing_pipeline()
+    try:
+        test.espn_cfb_pbp()
+        json_dict_stuff = test.run_processing_pipeline()
```

Ruff (0.14.7)

Local variable `json_dict_stuff` is assigned to but never used

Remove assignment to unused variable `json_dict_stuff`

(F841)